README.md

```diff
@@ -15,23 +15,35 @@
 ### Run app
 On IDEA create new gradle configuration and use `bootRun` as the run argument with env variables
```

Comment:
1. To find gradle configurations: From menu: Run -> Edit configurations...
2. How to make IDEA use the gradle conifguration?